r/rsparkling/tests/testthat/test_transforms.R

```diff
@@ -0,0 +1,100 @@
+library(rsparkling)
```

Comment:
This is required when running just R scripts in `tests`. If there is testthat.R then it will process all tests as a childs, so with testthat you can attach packages globally for testthat tests.